nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardProcessorNode.java

```diff
@@ -1283,7 +1285,7 @@ public Void call() throws Exception {
     /**
      * Will idempotently stop the processor using the following sequence: <i>
      * <ul>
-     * <li>Transition (atomically) Processor's scheduled state form RUNNING to
+     * <li>Transition (atomically) Processor's scheduled state from RUNNING to
      * STOPPING. If the above state transition succeeds, then execute the stop
      * task (asynchronously) where 'activeThreadMonitorCallback' provided by the
```

Comment:
same javadoc update needed here if switching away from the callback

https://github.com/markap14/nifi/blob/356eb41e2758df176d8142386316952c5d68c627/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/scheduling/StandardProcessScheduler.java#L325